shell/platform/windows/platform_handler_winuwp.cc

```diff
@@ -21,24 +28,81 @@ PlatformHandlerWinUwp::PlatformHandlerWinUwp(BinaryMessenger* messenger,
 
 PlatformHandlerWinUwp::~PlatformHandlerWinUwp() = default;
 
+bool isWindowInForeground() {
+  auto activation_mode =
+      winrt::Windows::UI::Core::CoreWindow::GetForCurrentThread()
+          .ActivationMode();
+  return activation_mode == winrt::Windows::UI::Core::CoreWindowActivationMode::
+                                ActivatedInForeground;
+}
+
 void PlatformHandlerWinUwp::GetPlainText(
     std::unique_ptr<MethodResult<rapidjson::Document>> result,
     std::string_view key) {
-  // TODO: Implement. See https://github.com/flutter/flutter/issues/70214.
-  result->NotImplemented();
+  // We call `Clipboard::GetContent()` when the application window is in
+  // focus, otherwise calling this will throw an error.
+  if (isWindowInForeground()) {
+    auto content =
+        winrt::Windows::ApplicationModel::DataTransfer::Clipboard::GetContent();
+    // Calling `DataPackageView::GetTextAsync()` when the clipboard has no text
+    // content will throw an error.
+    if (content.Contains(winrt::Windows::ApplicationModel::DataTransfer::
+                             StandardDataFormats::Text())) {
+      // Waiting `DataPackageView::GetTextAsync()` using `TResult.get()` on the
+      // platform thread will causes the application stop, so we continue
+      // response on this callback.
+      content.GetTextAsync().Completed([result = std::move(result), key](
+                                           auto async_info,
+                                           auto _async_status) {
+        auto clipboard_string = async_info.GetResults();
+
+        rapidjson::Document document;
+        document.SetObject();
+        rapidjson::Document::AllocatorType& allocator = document.GetAllocator();
+        document.AddMember(
+            rapidjson::Value(key.data(), allocator),
+            rapidjson::Value(Utf8FromUtf16(clipboard_string), allocator),
+            allocator);
+        result->Success(document);
+      });
+    } else {
+      result->Success(rapidjson::Document());
+    }
+  } else {
+    result->Success(rapidjson::Document());
```

Comment:
With the example, I found that `Clipboard.getData()` throws PlatformException when app is not focused, in web, but here Clipboard.getData() send success. I will fix it.

Reply:
done